constants/petnums.json

```diff
@@ -1,6 +1,6 @@
 {
   "GOLDEN_DRAGON": {
-    "COMMON": {
+    "LEGENDARY": {
```

Comment:
Since when are there multiple types of dragon pets

Reply:
there wasnt i just did a lil brain fart